src/dune_engine/action_to_sh.ml

```diff
@@ -55,6 +55,9 @@ let simplify act =
     | Ignore (outputs, act) ->
       Redirect_out (block act, outputs, Dev_null) :: acc
     | Progn l -> List.fold_left l ~init:acc ~f:(fun acc act -> loop act acc)
+    | Conc l ->
+      (* For now same as progn *)
```

Comment:
Could this be implemented with
`(a1 & a2 & a3 & a4 & wait)` ?

Reply:
I have implemented that, but I will wait for https://github.com/ocaml/dune/pull/7039 before pushing here. (I used the test to write it).

Reply:
Sounds good

Reply:
I've added a test for it in `expect-tests/dune_engine/action_to_sh_tests.ml`.